### ISSUE TITLE
Would like the ability to create an epic without child issues

### ISSUE THREAD

**Issue by mkwu** [waffleio/waffle.io#3588]
From a product planning perspective, sometimes we want to create an epic issue to track for planning and prioritization purposes and don't exactly know what children issues we need to create yet.

**Comment by adamzolyak** [comment]
Hi @mkwu!  Thanks for the feedback!  Totally understand why it's useful to be able to create an epic before you know exactly what children issues you need to create.  We'd suggest adding "Epic: " to the title or use an "epic" label.

**Comment by mkwu** [comment]
Would this allow me to filter by epic in your filters  and see these issues
along with the actual epic issues?  Doesn't seem like that is the case from
my testing.

**Comment by AnaMSteele** [comment]
@mkwu you are correct from your testing. You would need to have the epic label on both the epic parent, epic child, and also the holding place epic issues you were creating. We do not have this on our short term roadmap, but we are tracking in our planned column the work to include "OR" in our filter options, which would allow you to label placeholders with epic and create a filter that shows those, as well as existing epics and their children.  That work is being tracked on #1440, and is on our roadmap for the near term.

**Comment by janaagaard75** [comment]
I think adding an "epic" label sounds like a good solution, but only if Waffle starts treating the label a special one: Issues with this label should be shown when only showing epics on the board and features with this label should (probably?) get the little blue bookmark icon next to the number.
